android/app/src/main/java/app/priceguard/ui/detail/ProductDetailViewModel.kt

```diff
@@ -81,15 +82,15 @@ class ProductDetailViewModel @Inject constructor(
 
     fun getDetails(isRefresh: Boolean) {
         viewModelScope.launch {
-            if (::productCode.isInitialized.not()) {
+            if (::productCode.isInitialized.not() && _state.value.shop == null) {
```

Comment:
여기 조건문 하나라도 빠지면 안되는 조건이 아닌가 싶어서 적어둡니다.
```suggestion
            if (::productCode.isInitialized.not() && ::productShop.isInitialized.not()) {
```

Reply:
저도 이렇게 변경하는게 더 좋아보입니다

Reply:
수정했습니다!